src/openrct2-ui/windows/Guest.cpp

```diff
@@ -2175,3 +2039,76 @@ void window_guest_inventory_paint(rct_window* w, rct_drawpixelinfo* dpi)
         }
     }
 }
+
+/**
+ *
+ *  rct2: 0x00698315
+ */
+void window_guest_debug_update(rct_window* w)
+{
+    w->frame_no++;
+    window_invalidate(w);
+}
+
+static void draw_debug_label(rct_drawpixelinfo* dpi, int32_t x, int32_t y, const char* label, const char* value)
+{
+    const char* args[2] = { label, value };
+    gfx_draw_string_left(dpi, STR_PEEP_DEBUG_LABEL, args, 0, x, y);
+}
+
+void window_guest_debug_paint(rct_window* w, rct_drawpixelinfo* dpi)
+{
+    window_draw_widgets(w, dpi);
+    window_guest_overview_tab_paint(w, dpi);
+    window_guest_stats_tab_paint(w, dpi);
+    window_guest_rides_tab_paint(w, dpi);
+    window_guest_finance_tab_paint(w, dpi);
+    window_guest_thoughts_tab_paint(w, dpi);
+    window_guest_inventory_tab_paint(w, dpi);
+    window_guest_debug_tab_paint(w, dpi);
+
+    auto peep = GET_PEEP(w->number);
+    auto x = w->x + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].left + 4;
+    auto y = w->y + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].top + 4;
+
+    char buffer[512]{};
+    snprintf(buffer, sizeof(buffer), "%i, %i, %i", peep->x, peep->y, peep->z);
```

Comment:
As I mentioned before, I really want these to be proper localised strings. We did that with all other debugging tools as well.

Reply:
ok, done